autoflake.py

```diff
@@ -569,9 +607,11 @@ def _main(argv, standard_out, standard_error):
                         help='by default, only unused standard library '
                              'imports are removed; specify a comma-separated '
                              'list of additional modules/packages')
+    parser.add_argument('--expand-single-star-import', action='store_true',
```

Comment:
Well, I think if we want to describe it in the option name, `single` is better from the meaning perpective http://wikidiff.com/lone/single 😄